src/cmd/run.go

```diff
@@ -481,6 +488,10 @@ func constructExecArgs(container string,
 
 	execArgs = append(execArgs, envOptions...)
 
+	if runFlags.preserveFDs != 0 {
```

Comment:
I suspect that this should be `> 0` because that's also what [Podman](https://github.com/containers/podman/commit/0b34327ad40e04861dac7f73870d87633a5c637e) uses.  Otherwise, we will only error out later from `podman exec ...` itself:
```
$ toolbox run --preserve-fds -12 true
Error: failed to invoke 'podman exec' in container fedora-toolbox-36
$ echo $?
125
```

This makes me wonder if we should be using an unsigned integer flag.  :)

Reply:
I see that newer versions of Podman [do use](https://github.com/containers/podman/commit/4d895dcb5472da19b886ca1662182556242fe5d4) an unsigned integer flag.

*It's not obvious from the commit message, but that's where it changed.*

Reply:
Damn!  Converting an *unsigned* integer to string is surprisingly annoying.  :)

[strconv.Itoa](https://pkg.go.dev/strconv#Itoa) takes a *signed* `int`, which would require a cast, and there's no unsigned counterpart.  There's [strconv.FormatUint](https://pkg.go.dev/strconv#FormatUint) which takes an unsigned `uint64`, which is better, but would still require a cast.

So, [fmt.Sprint](https://pkg.go.dev/fmt#Sprint) it is, if we want to avoid the casting.  It's a bit more expensive than the other two functions, but we don't have to worry about it unless it's proven to be a performance bottle neck.